### ISSUE TITLE
Move LongPollingMockConsumer to main artefact

### ISSUE THREAD

**Issue by astubbs** [confluentinc/parallel-consumer#526]
It's too difficult to use as test artefact.

**Comment by cykl** [comment]
Any update on this? 

Having to copy paste this code in each project and maintain it is annoying. It would be great if clients could just depend on it.

**Comment by astubbs** [comment]
Tracked in the maintained fork as astubbs/parallel-consumer#159, which carries a summary of this
issue and its current status there.

The **Confluent-hosted version** of this project is no longer maintained - its README says so.
**Parallel Consumer itself continues** in my fork: https://github.com/astubbs/parallel-consumer

Nothing in this repository is closed or changed by this comment.

**Comment by astubbs** [comment]
@cykl - done and released in the maintained fork, in [v0.6.0.0](https://github.com/astubbs/parallel-consumer/releases/tag/v0.6.0.0). The coordinates changed with the fork - on Maven Central it is now:

```xml
<dependency>
    <groupId>bz.stub.parallelconsumer</groupId>
    <artifactId>parallel-consumer-core</artifactId>
    <version>0.6.0.0</version>
</dependency>
```

(`parallel-consumer-vertx`, `parallel-consumer-reactor` and `parallel-consumer-mutiny` likewise.) The Java package is `bz.stub.parallelconsumer` too, so imports change with it - one command rewrites them (GNU `sed`; on macOS use `sed -i ''`):

```bash
find . -name '*.java' -exec sed -i 's/io\.confluent\.parallelconsumer/bz.stub.parallelconsumer/g' {} +
```

The short version: `LongPollingMockConsumer` ships in the main `parallel-consumer-core` artefact, so a broker-free test of an integration no longer needs the `tests` classifier jar (astubbs/parallel-consumer#202). The fork's mirror is astubbs/parallel-consumer#159.
